dask/array/tests/test_creation.py

```diff
@@ -571,9 +571,15 @@ def test_repeat():
         assert all(concat(d.repeat(r).chunks))
 
 
+@pytest.mark.parametrize("a", [da.asarray([0, 1, 2]), [[1, 2], [3, 4]]])
+@pytest.mark.parametrize("reps", [2, (2, 2), (1, 2), (2, 1), (2, 3, 4, 0)])
+def test_tile_basic(a, reps):
+    assert_eq(np.tile(a, reps), da.tile(a, reps))
```

Comment:
As below, we'll need to make sure that the argument to `np.tile` is a numpy array.

Reply:
Changed now.